binaries/cli/src/up.rs

```diff
@@ -28,6 +28,20 @@ pub(crate) fn up(config_path: Option<&Path>) -> eyre::Result<()> {
 
     if !daemon_running(&mut *session)? {
         start_daemon().wrap_err("failed to start dora-daemon")?;
+
+        // wait a bit until daemon is connected
+        let mut i = 0;
+        const WAIT_S: f32 = 0.1;
+        loop {
+            if daemon_running(&mut *session)? {
+                break;
+            }
+            i += 1;
+            if i > 20 {
```

Comment:
```suggestion
            if i % 20 == 0 {
```
To not raise a wall of warnings after 2 seconds.

Reply:
Right now we abort with an error, so it would occur only once.

Reply:
Indeed, I thought, it was a warning.